go.mod

```diff
@@ -1,12 +1,16 @@
 module github.com/cosmtrek/air
 
-go 1.16
+go 1.17
 
 require (
-	github.com/creack/pty v1.1.11
```

Comment:
`github.com/creack/pty` was removed when `go mod tidy` was run because it was not used in any way.